web/src/settings_account.js

```diff
@@ -774,9 +775,28 @@ export function set_up() {
     $("#user_timezone").on("change", function (e) {
         e.preventDefault();
         e.stopPropagation();
+        const chosen_timezone = this.value;
+        const data = {timezone: chosen_timezone};
+        if (chosen_timezone !== timerender.display_time_zone) {
```

Comment:
We need to canonicalize `chozen_timezone` for this test. (It may have been server-canonicalized but it is not browser-canonicalized.)

The `if` can be simplified away:

```js
data.web_suggest_update_timezone = chosen_timezone === timerender.display_time_zone;
```

Reply:
Done